cardano-node/src/Cardano/Node/Parsers.hs

```diff
@@ -46,6 +46,7 @@ nodeRunParser = do
   topFp <- lastOption parseTopologyFile
   dbFp <- lastOption parseDbPath
   socketFp <- lastOption $ parseSocketPath "Path to a cardano-node socket"
+  fwdSocketFp <- lastOption $ parseForwardSocketPath "Path to a forward socket"
```

Comment:
Can we elaborate a little more on `"Path to a forward socket"`? Maybe `"Path to a trace-dispatcher forwarding socket"`

Reply:
That's an outdated bit of code -- it's now documented as:
- `"Accept incoming cardano-tracer connection at local socket"`, and
- `"Connect to cardano-tracer listening on a local socket"`

Reply:
@deepfire Strictly speaking, this socket is for forwarding to _any_ external Haskell program, and `cardano-tracer` is just one example of such a program. So, this socket doesn't know anything about `cardano-tracer`. :-)